tests/bin/test_cli.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import re
+from collections import Counter
+from unittest import TestCase
+
+from airflow.bin.cli import CLIFactory
+
+P_LONG_OPTION_ARG = re.compile("^--\\w{2}")
+P_SHORT_OPTION_ARG = re.compile("^-\\w$")
+
+
+class TestCli(TestCase):
```

Comment:
I love the tests!

Reply:
Hi,Jarek should we change long option name style? from `snake_case` to `kebab-case`, for example using `airflow tasks test --dry-run` instead of `airflow tasks test --dry_run`, I start a [discuss] in dev mail list to discuss it https://lists.apache.org/thread.html/r95c53953499a236466e0b762dfdadd5ca2ba9d6e2a3516c699a14380%40%3Cdev.airflow.apache.org%3E  WDYT

Reply:
Yeah. I missed that one. You are completely right kebab-case is much better (I did not even know the name :) ).

Reply:
So I will change our long option name, BTW, I just know that name(kebab-case), day ago when I search the stackoverflow ;-)